src/Patch/Document.js

```diff
@@ -71,6 +71,16 @@ export default function(internals) {
       return result;
     });
 
+  Object.defineProperty(document, 'body', {
```

Comment:
I didn't use `Document.prototype` because that didn't work. Now I found out why: `body` is actually defined on `HTMLDocument.prototype`. I'll update the PR accordingly.